timing_bench_test.go

```diff
@@ -46,7 +47,9 @@ func BenchmarkChannel(b *testing.B) {
 	b.ResetTimer()
 	for i := 0; i < concurrency; i++ {
 		go func() {
-			ch <- &timing{host: "myhost", dur: dur}
+			for j := 0; j < int(math.Ceil(float64(b.N)/concurrency)); j++ {
```

Comment:
@myleshorton The reason why you couldn't get the benchmark to register a non-zero number is that you ignored the `b.N` parameter. That parameter determines the number of "ops" that the benchmark is expecting to run, and it calculates the time per op by dividing the runtime by `b.N`.

After adding back in code to make sure we actually run approximately `b.N` iterations, we get these results:

```
➜  httpseverywhere git:(simpler) ✗ go test -v -run blah -bench . | grep -v "^DEBUG"
BenchmarkNoMatch-4   	DEBUG httpse: httpse.go:155 Average running time: 0ms
   30000	     49316 ns/op
BenchmarkChannel-4   	 2000000	       626 ns/op
BenchmarkLock-4      	DEBUG channel: timing_bench_test.go:32 Average running time: 0ms
   30000	     47065 ns/op
PASS
ok  	github.com/getlantern/httpseverywhere	12.987s
```

As expected, simply submitting new stat structs to the channel is pretty cheap.